app/src/main/java/org/bostwickenator/ftpuploader/MainActivity.java

```diff
@@ -149,6 +156,11 @@ public void run() {
                     boolean result = ftp.storeFile(file);
                     if(result) {
                         uploadRecordDatabase.addFile(file);
+                        if (settingsStore().getBoolean(SETTING_DELETE_AFTER_UPLOAD, false)) {
+                            if (!file.delete()) {
+                                Logger.error("failed deleting file");
```

Comment:
Is it possible to provide the logger with which file failed to delete? 
Would help with troubleshooting for the user.

Reply:
Good idea. Done in 29f3d1edc85efd0c6abe8a104f76899b3a6207e2